pkgs/applications/networking/soju/default.nix

```diff
@@ -0,0 +1,38 @@
+{ lib, buildGoModule, fetchFromSourcehut, installShellFiles, scdoc }:
+
+buildGoModule rec {
+  pname = "soju";
+  version = "0.1.2";
+
+  src = fetchFromSourcehut {
+    owner = "~emersion";
+    repo = "soju";
+    rev = "v${version}";
+    sha256 = "sha256-dauhGfwSjjRt1vl2+OPhtcme/QaRNTs43heQVnI7oRU=";
+  };
+
+  vendorSha256 = "sha256-0JLbqqybLZ/cYyHAyNR4liAVJI2oIsHELJLWlQy0qjE=";
+
+  subPackages = [
+    "cmd/soju"
+    "cmd/sojuctl"
+    "contrib/znc-import.go"
+  ];
+
+  nativeBuildInputs = [
+    scdoc
+    installShellFiles
+  ];
+
+  postInstall = ''
+    scdoc < doc/soju.1.scd > doc/soju.1
+    installManPage doc/soju.1
+  '';
+
+  meta = with lib; {
+    description = "A user-friendly IRC bouncer";
+    homepage = "https://soju.im";
+    license = licenses.agpl3Only;
+    maintainers = with maintainers; [ malvo ];
```

Comment:
Please add yourself as a maintainer in `lib/maintainers` and make this into a separate commit:

```
maintainers: add <name>
```

Reply:
```
error: undefined variable 'malvo'

       at /home/jon/.cache/nixpkgs-review/pr-123815/nixpkgs/pkgs/applications/networking/soju/default.nix:36:39:

           35|     license = licenses.agpl3Only;
           36|     maintainers = with maintainers; [ malvo ];
             |                                       ^
           37|   };
```

Reply:
Oops, thought I already did that ._.